### DISCUSSION TITLE
Amazon Lex

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS3073-C/Python-Community-GS3073-C#385] [General]
不同於flask，Amazon有推出自己的聊天機器人建構平台—Amazon Lex。
許多粉絲專業也會應用在自動回覆訊息上，或是達成某些條件(ex. IG分享、按讚)，就自動送某種禮包等等，大家應該都有在生活上看過，我之前有參加過AWS舉辦的工作坊，以下是當時的簡報節錄。
**SaaS**就像是Gmail平台，可以直接應用
**PaaS**與**IaaS**都是雲端服務平台，不同之處在於前者的已經有建構好的基礎，只需要輸入邏輯，而後者連基礎也可以自己決定
**Amazon Lex**是屬於**PaaS**
我覺得Amazon Lex可以很方便的建構，就算是自學也很容易上手! 有興趣可以去玩玩看 
<img width="960" alt="amazon" src="https://user-images.githubusercontent.com/90574174/173119652-865a8fd2-0786-44c6-ad08-2f87a5e8517d.png">

**Comment by luo910227** [comment]
我覺得已有建構基礎的雲端服務平台應該挺值得學習的，如果能夠學會怎麼操作達成某些條件就自動推送事件的功能也挺有趣的，感謝你的介紹
